README.md

```diff
@@ -27,23 +27,34 @@ Startup: `rake servers:start`
 
 ## How to start application locally 
 
-Run the `bin/rails server` command, then in a browser connect to [localhost:3000](http://localhost:3000/)
+Run the `bin/rails server` or `rails s` command, then in a browser connect to [localhost:3000](http://localhost:3000/)
 
 ## How to run the test suite
 
-`bundle exec rspec spec`
+`bundle exec rspec <spec/**/*>`
 
 ## Deploying 
 
-Currently, this application can not be deployed with pulbot. You must use Capistrano on the command line. 
+You may use Capistrano on the command line. 
 
 `BRANCH=branch_name bundle exec cap staging deploy`
 
+`BRANCH=branch_name bundle exec cap production deploy`
+
+Alternatively, you may deploy from [ansible-tower](https://ansible-tower.princeton.edu/).
+
 ## How to load data 
 
-We want to load in CSV files that contain GuideCards and SubGuides data, which was exported from the legacy version of this application. The data lives in the `data` folder of this project. 
+We want to load in CSV files that contain GuideCard and SubGuideCard data, which was exported from the legacy version of this application. The data lives in the `data` folder of this project. 
+
+To list all import services for the application: `rake -T | grep import`
+
+To import the GuideCard records: `rake import:import_guide_cards`
+To import the SubGuideCard records: `rake import:import_sub_guide_cards`
+
+The CardImage records are the images that are included in the GuideCard and SubGuideCard records. These will take the longest to import (totaling ~1.5 million images).
 
-To import the GuideCards records: `rake import:import_guide_cards`
+To import the CardImage records: `rake import:import_card_images`
```

Comment:
I suspect this requires some aws profile configuration; maybe that's worth noting here and fleshing out in the future when someone else tries to do it?

Reply:
Hi Anna, 

I have updated the file with the requested changes. I included some instructions on setting up aws for a local machine (instructions based on [onboarding.md](https://github.com/pulibrary/rdss-handbook/blob/2d571795873730fffc95fb8b94c3aa6fc5977d2c/onboarding.md?plain=1#L8)). Do you think I should just leave it as a note, that aws setup is required, or that these instructions are detailed enough. Thanks!